src/components/organisms/AuthenticationModal/RegisterForm/RegisterForm.tsx

```diff
@@ -53,7 +58,12 @@ const RegisterForm: React.FunctionComponent<PropsType> = ({
   closeAuthenticationModal,
 }) => {
   const history = useHistory();
-  const venue = useSelector(venueSelector);
+  const venueFromSelector = useSelector(venueSelector);
```

Comment:
This is old/legacy, you can remove it and just use the value from `useConnectCurrentVenueNG`

Reply:
Not an issue for the display of `NotFound` if value comes from old selectors.

[OOS] Trying to be less disruptive with this PR so an emergent issues will be fixed quicker.
Same selector logic is used in login, just like the registration form, so updating those should be done in separate, refactoring, PR.

Reply:
We should at least add `@debt` to it and explanation, otherwise it will fly away and no one will remember to fix it.

Reply:
Since now the loading and flashes fix are in staging, I can probably do this as well.

Reply:
updated @ 72c493b88f7f0ff2349151449c4a0636634b4880